internal/weldr/api.go

```diff
@@ -82,23 +83,26 @@ func (cs ComposeState) ToString() string {
 // systemRepoIDs returns a list of the system repos
 // NOTE: The system repos have no concept of id vs. name so the id is returned
 func (api *API) systemRepoNames() (names []string) {
-	for _, repo := range api.repos {
-		names = append(names, repo.Name)
+	repos, err := api.repoRegistry.ReposByArch(api.arch, false)
```

Comment:
I would like to discuss, whether the image-type-specific repos should be treated as System Repos. Currently they are not and @bcl didn't want to treat them in this way. However as a result of not treating them this way, the user will be able to define sources with the same name as the image-type-specific repository. I'm not sure if this will be a problem or not.

Reply:
I think those repos make sense as a separate namespace that users don't see and cannot touch.
The whole system repo thing is an artifact of how lorax-composer worked, combining the host repos with custom user repos -- the system repos came from /etc/yum.repos.d/ at startup, and the user repos were stored in lorax-composer. Now that we don't depend directly on the host repo files I think it makes sense to hide anything the user cannot change since there's nothing they can do with them anyway.

Reply:
@bcl hmm, interesting thought. Hiding anything the user can not change would actually imply also hiding the "system" repositories. Is that what you are suggesting?

Reply:
yes, but I suppose we should ask someone working on the UI if they think it is important to show them.

Reply:
Even though the user cannot change the default repos I think it is still good to show the user where the packages are being pulled from.